specification/ingest/_types/Processors.ts

```diff
@@ -348,6 +354,60 @@ export class AttachmentProcessor extends ProcessorBase {
   resource_name?: string
 }
 
+export class GeoGridProcessor extends ProcessorBase {
+  /**
+   * The field to interpret as a geo-tile.=
+   * The field format is determined by the `tile_type`.
+   */
+  field: string
+  /**
+   * Three tile formats are understood: geohash, geotile and geohex.
+   */
+  tile_type: GeoGridTileType
+  /**
+   * The field to assign the polygon shape to, by default, the `field` is updated in-place.
+   * @server_default field
+   */
+  target_field?: Field
+  /**
+   * If specified and a parent tile exists, save that tile address to this field.
+   */
+  parent_field?: Field
+  /**
+   * If specified and children tiles exist, save those tile addresses to this field as an array of strings.
+   */
+  children_field?: Field
+  /**
+   * If specified and intersecting non-child tiles exist, save their addresses to this field as an array of strings.
+   */
+  non_children_field?: Field
+  /**
+   * If specified, save the tile precision (zoom) as an integer to this field.
+   */
+  precision_field?: Field
+  /**
+   * If `true` and `field` does not exist, the processor quietly exits without modifying the document.
+   * @server_default false
+   */
+  ignore_missing?: boolean
+  /**
+   * Which format to save the generated polygon in.
+   * @server_default GeoJSON
+   */
+  target_format?: GeoGridTargetFormat
+}
+
+export enum GeoGridTileType {
+  geotile,
+  geohex,
+  geohash
+}
+
+export enum GeoGridTargetFormat {
+  geojson,
+  wkt
```

Comment:
While the documentation says it should be `GeoJSON` and `WKT`, the YAML tests are using `geojson` and `wkt`. I talked to @craigtaverner who confirms that this processor was specifically designed to be case-insensitive so lowercase is OK.